### DISCUSSION TITLE
Does `traceSamplingPolicySettingsFile` need to configure absolute path?

### DISCUSSION THREAD

**Discussion by linghengqian** [apache/skywalking#9010] [Q&A] [answered]
- The default value of `agent-analyzer.default.traceSamplingPolicySettingsFile` is `trace-sampling-policy-settings.yml`. It can be found directly in the `config` folder without the need to configure an absolute path? I see the working directory set by the `Dockerfile` is `/skywalking` not `/skywalking/config`. I'm guessing the documentation is actually referring to `/skywalking/config/trace-sampling-policy-settings.yml` instead of `./trace-sampling-policy-settings.yml`?

- Refer to https://skywalking.apache.org/docs/main/v9.0.0/en/setup/backend/trace-sampling/#set-the-sample-rate . 

- ![image](https://user-images.githubusercontent.com/20187731/167283920-6abbb71a-0d13-4e5f-b55c-aa42d06a016f.png)

**Comment by wu-sheng** [comment, marked as answer]
I am not sure why do you ask this? Java codes use classpath to load resource files, not `working directory`

**Reply by linghengqian** [reply]
I misunderstood the file loading method here, thank you for your answer, classpath does not need to care about the working directory.
